wdio.conf.js

```diff
@@ -156,7 +156,7 @@ exports.config = {
         return;
       }
       var message = assertion.error.message.replace(/[^a-z0-9]/gi, '_').toLowerCase();
-      var location = `${__dirname}/test_reports/e2e/assertionError_` + message + '.png';
+      var location = `${__dirname}/test_reports/e2e/screenshot/assertionError_` + message + '.png';
```

Comment:
This line uses both interpolation and concatenation. Should do only one
```suggestion
      var location = `${__dirname}/test_reports/e2e/screenshot/assertionError_${message}.png`;
```